winterfell/services/claimService.js

```diff
@@ -92,16 +95,16 @@ module.exports.findIncompleteClaimOrCreate = function(userId, forms, callback) {
   });
 };
 
-module.exports.addForm = function(claimId, file, callbacks) {
+module.exports.addForm = function (claimId, file, callbacks) {
```

Comment:
I have no idea what is standard and what is not in javascript TBH, once we turn on the linter it should all be taken care of and standardized.